src/main/java/com/amazon/janusgraph/graph/AbstractGraphFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.amazon.janusgraph.graph;
+
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+
+/**
+ * Created by addisonslabaugh on 6/22/17.
+ */
+public interface AbstractGraphFactory {
```

Comment:
not sure this class is necessary, maybe you can add this logic to the test you added to ScenarioTests